DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+describe('login', () => {
+
+    const testData = require('../fixtures/testData.json')[0]
+    beforeEach(() => {
+        cy.visit("https://demoqa.com/automation-practice-form")
+    });
+    it('submit successfully with valid input', () => {
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get('#userEmail').type(data.userEmail)
+            cy.get("label[for='gender-radio-1']").click();
+            cy.get("#userNumber").type(data.userNumber)
+            cy.get('#dateOfBirthInput').click()
+            cy.get('.react-datepicker__month-select').select('January')
```

Comment:
Have a look at `Date()` in JS to deal with date time. Also see:
- [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)
- [toLocaleDateString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)

Try to make your code more flexible